lib/spack/docs/developer_guide.rst

```diff
@@ -407,15 +407,15 @@ the string that it detected to be the name and version. The
 print URLs that were not being parsed correctly.
 
 """"""""""""""""""
-``spack url test``
+``spack url summary``
```

Comment:
I also keep running `spack list -cen` instead of `spack url list -cen`. But I can't think of a better name for that one.